taxi/test_forms.py

```diff
@@ -0,0 +1,114 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from taxi.forms import (
+    CarForm,
+    DriverCreationForm,
+    DriverLicenseUpdateForm,
+    validate_license_number,
+)
+from taxi.models import Car, Manufacturer
+from django.core.exceptions import ValidationError
```

Comment:
Please, sort imports and add blank lines between different groups according to PEP8